From sd@sedf.de  Fri Nov  6 14:57:06 2009
From: Sebastian Dalfuﬂ <sd@sedf.de>
Date: Fri, 6 Nov 2009 14:49:00 +0100
Subject: Staging: rt2860: remove remainders of /etc reading stuff
To: greg@kroah.com
Message-ID: <20091106134900.GB27210@sedf.de>

From: Sebastian Dalfuﬂ <sd@sedf.de>

The stuff that tries to read a file from /etc is already removed, so
this patch just removes the last remainders.

Signed-off-by: Sebastian Dalfuﬂ <sd@sedf.de>
Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
---
 drivers/staging/rt2860/TODO       |    1 -
 drivers/staging/rt2860/rt_linux.h |    2 --
 2 files changed, 3 deletions(-)

--- a/drivers/staging/rt2860/rt_linux.h
+++ b/drivers/staging/rt2860/rt_linux.h
@@ -80,11 +80,9 @@
  ***********************************************************************************/
 
 #ifdef RTMP_MAC_PCI
-#define STA_PROFILE_PATH			"/etc/Wireless/RT2860STA/RT2860STA.dat"
 #define STA_DRIVER_VERSION			"2.1.0.0"
 #endif /* RTMP_MAC_PCI // */
 #ifdef RTMP_MAC_USB
-#define STA_PROFILE_PATH			"/etc/Wireless/RT2870STA/RT2870STA.dat"
 #define STA_DRIVER_VERSION			"2.1.0.0"
 /* RT3070 version: 2.1.1.0 */
 #endif /* RTMP_MAC_USB // */
--- a/drivers/staging/rt2860/TODO
+++ b/drivers/staging/rt2860/TODO
@@ -9,7 +9,6 @@ TODO:
 	- checkpatch.pl clean
 	- sparse clean
 	- port to in-kernel 80211 stack and common rt2x00 infrastructure
-	- remove reading from /etc/ config files
 	- review by the wireless developer community
 
 Please send any patches or complaints about this driver to Greg
